[TW-1188] Break up "Private API Network" (#4910)
postmanlabs/postman-docs: src/pages/docs/collaborating-in-postman/private-api-network/adding-private-network.md

You can search and sort elements and folders from the left sidebar.

<img alt="Create new folder in Private Network" src="https://assets.postman.com/postman-docs/v10/private-api-network-organize-folders-v10-2.jpg" width="300px"/>

Select <img alt="Home icon" src="https://assets.postman.com/postman-docs/v10/icon-home-v10.jpg#icon" width="16px"> **Home** in the left sidebar to search, filter, and sort all elements in your Private API Network. Select a folder in the left sidebar to search, filter, and sort elements only in the selected folder.

* To search by name, enter the full or partial element or folder name into **Search elements and folders**.
* To sort elements and folders, select the sort icon <img alt="Sort icon" src="https://assets.postman.com/postman-docs/icon-sort.jpg#icon" width="16px">. Options are:
    * **Default** - Sort by type, grouping folders and elements by type.
    * **Date added** - Sort by the date that elements and folders were added.

>     * **A to Z** - Sort elements and folders alphabetically.

Reviewer: This section is meant to explain how to use the search bar and sort button (to the right of it) in the left sidebar. The search bar and sort button sort the folders and elements that appear in the left sidebar.

We can definitely rework this section, but I think the paragraph on line 70 isn't related to the bullet point options on lines 72-76.

Author: Thanks for catching that; I think that was a copy-paste mistake.

Select a folder from the left sidebar to search, filter, and sort its elements in the folder overview on the right.

<img alt="Private API List" src="https://assets.postman.com/postman-docs/v10/private-api-network-list-v10-2.jpg"/>

* To search by name, enter the full or partial element name into **Search elements**.
* To filter based on the person who added the element, select **Added by**, then select the name of the team member.
* To filter by element type, select **Type**, then select **API**, **Collection**, or **Workspace**.
* To filter based on element tags, select **Tags**, then select the tags. Learn more about adding tags to [collections](/docs/collections/using-collections/#tagging-a-collection), [APIs](/docs/designing-and-developing-your-api/managing-apis/#tagging-apis), and [workspaces](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#tagging-a-workspace).
* To sort elements, select the **Sort by** dropdown list. Options are:
    * **Recently added** - Sort elements by the date they were added.
    * **A to Z** - Sort elements alphabetically.
    * **Trending this week** - Sort elements by your team's usage this week.

>     * **Trending this month** - Sort elements by your team's usage this month.

Reviewer: This section is meant to explain how to search and filter elements using the options in the bullet points. Users can select either Home or a folder (in the left sidebar) to search /filter elements on the right.

Users can select Home to search and filter all elements in the network. Users can also select a folder to search and filter elements in that folder.

We can definitely rework this, but I think we should also add the detail about selecting Home.

Author: What about reworking it so it's broken up into a h4 sub-section like this:
![image](https://github.com/postmanlabs/postman-docs/assets/113702138/6ab8223c-a442-4af8-9e65-7288c1e9666b)

Reviewer: Yeah, I like the idea of giving these sections headers!

Also, I'm wondering if we should bring the current H3 headers up to H2 headers. Now that the content has its own page, I'm wondering if we need the same amount of levels.

Author: That's a good point; I'll update the headers!

  - type: link
    name: "Introducing an API to manage your Private API Network with more automation"
    url: "https://blog.postman.com/introducing-api-to-manage-your-private-api-network-with-automation/"
  - type: link
    name: "Simplifying API distribution and evaluation with the Private API Network"
    url: "https://blog.postman.com/simplifying-api-distribution-and-evaluation-with-the-private-api-network/"
  - type: subtitle
    name: "Case Studies"
  - type: link
    name: "ChargeHub plans to improve internal discovery with a Private API Network"
    url: "https://www.postman.com/case-studies/chargehub/"

---

> **[The Private API Network is available on Postman Enterprise plans.](https://www.postman.com/pricing)**

The _Private API Network_ provides a [central directory](https://www.postman.com/api-platform/api-catalog/) of workspaces, collections, and APIs your team uses internally. Your Postman team can access these resources and start using them right away. By using the Private API Network, you can enable developers across your organization to discover, consume, and track API development in one place.

> Workspaces, collections, and APIs in the Private API Network are visible to logged-in users who are on your Postman team. Users who aren't on your team can't find or access these resources. [Super Admins](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) and [API Network Managers](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) can manage all folders and elements in your Private API Network.

Reviewer: Since this page is an overview of the section, what do you think about also mentioning Folder Managers?

Author: I think that's a good idea. I'll update it.

<img alt="Private API Network overview" src="https://assets.postman.com/postman-docs/v10/private-api-network-overview-v10-3.jpg"/>

## Contents

* [Navigating the Private API Network](#navigating-the-private-api-network)
* [Searching, filtering, and sorting](#searching-filtering-and-sorting)
* [Reviewing details about elements](#reviewing-details-about-elements)
* [Sharing folders and elements](#sharing-folders-and-elements)
* [Watching and forking elements](#watching-and-forking-elements)
* [Private API Network reports](#private-api-network-reports)

## Navigating the Private API Network

The Private API Network is a good place to learn about workspaces, collections, and APIs shared within your team. Under your team name, you can browse a directory of elements shared within your team in the left sidebar.

There are two ways to access the Private API Network:

* Select **Home** from the Postman header, then select **Private API Network** in your team information on the left sidebar.
* Select **API Network** from the Postman header, then select **Private API Network**.

> ## Searching, filtering, and sorting
> 
> There are several ways to search, filter, and sort elements and folders in the Private API Network. You can search and sort all elements and folders from the left sidebar. You can also select a folder to search, filter, and sort elements on the right.
> 
> You can search and sort elements and folders from the left sidebar.
> 
> <img alt="Create new folder in Private Network" src="https://assets.postman.com/postman-docs/v10/private-api-network-organize-folders-v10-2.jpg" width="300px"/>
> 
> Select <img alt="Home icon" src="https://assets.postman.com/postman-docs/v10/icon-home-v10.jpg#icon" width="16px"> **Home** in the left sidebar to search, filter, and sort all elements in your Private API Network.
> 
> * To search by name, enter the full or partial element or folder name into **Search elements and folders**.
> * To sort elements and folders, select the sort icon <img alt="Sort icon" src="https://assets.postman.com/postman-docs/icon-sort.jpg#icon" width="16px">. Options are:
>     * **Default** - Sort by type, grouping folders and elements by type.
>     * **Date added** - Sort by the date that elements and folders were added.
>     * **A to Z** - Sort elements and folders alphabetically.
> 
> ### Search using the Private API Network overview
> 
> You can search, filter, and sort Private API Network folders as well as elements with the overview feature.
> 
> * Select <img alt="Home icon" src="https://assets.postman.com/postman-docs/v10/icon-home-v10.jpg#icon" width="16px"> **Home** in the left sidebar to search, filter, and sort all elements in your Private API Network.
> * Select a folder from the left sidebar to search, filter, and sort its elements in the folder overview on the right.
> 
> <img alt="Private API List" src="https://assets.postman.com/postman-docs/v10/private-api-network-list-v10-2.jpg"/>
> 
> The **All APIs, collections and workspaces** section in the overview lets you search, sort, and filter with the following options:
> 
> * To search by name, enter the full or partial element name into **Search elements**.
> * To filter based on the person who added the element, select **Added by**, then select the name of the team member.
> * To filter by element type, select **Type**, then select **API**, **Collection**, or **Workspace**.
> * To filter based on element tags, select **Tags**, then select the tags. Learn more about adding tags to [collections](/docs/collections/using-collections/#tagging-a-collection), [APIs](/docs/designing-and-developing-your-api/managing-apis/#tagging-apis), and [workspaces](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#tagging-a-workspace).
> * To sort elements, select the **Sort by** dropdown list. Options are:
>     * **Recently added** - Sort elements by the date they were added.
>     * **A to Z** - Sort elements alphabetically.
>     * **Trending this week** - Sort elements by your team's usage this week.
>     * **Trending this month** - Sort elements by your team's usage this month.

Reviewer: ```suggestion
## Searching, filtering, and sorting

There are several ways to search, filter, and sort elements and folders in the Private API Network. You can search and sort all elements and folders from the left sidebar. You can also select <img alt="Home icon" src="https://assets.postman.com/postman-docs/v10/icon-home-v10.jpg#icon" width="16px"> **Home** or a folder to search, filter, and sort elements on the right.

### Searching for folders and elements in the network

You can search and sort elements and folders in the network from the left sidebar. Then you can select a folder or element to view its details on the right.

<img alt="Create new folder in Private Network" src="https://assets.postman.com/postman-docs/v10/private-api-network-organize-folders-v10-2.jpg" width="300px"/>

* To search by name, select **Search elements and folders** and enter the full or partial element or folder name.
* To sort elements and folders, select the sort icon <img alt="Sort icon" src="https://assets.postman.com/postman-docs/icon-sort.jpg#icon" width="16px">. Options are:
    * **Default** - Sort by type, grouping folders and elements by type.
    * **Date added** - Sort by the date that elements and folders were added.
    * **A to Z** - Sort elements and folders alphabetically.

### Searching for elements in a folder

You can select a folder to search, filter, and sort elements on the right. Select <img alt="Home icon" src="https://assets.postman.com/postman-docs/v10/icon-home-v10.jpg#icon" width="16px"> **Home** in the left sidebar to search, filter, and sort all elements in your Private API Network. Select a folder in the left sidebar to search, filter, and sort elements only in the selected folder.

You can select a folder in the left sidebar to then search, filter, and sort elements on the right.

* Select <img alt="Home icon" src="https://assets.postman.com/postman-docs/v10/icon-home-v10.jpg#icon" width="16px"> **Home** in the left sidebar to search, filter, and sort all elements in your Private API Network on the right.
* Select a folder from the left sidebar to search, filter, and sort its elements on the right.

<img alt="Private API List" src="https://assets.postman.com/postman-docs/v10/private-api-network-list-v10-2.jpg"/>

The **All APIs, collections and workspaces** section enables you to search, sort, and filter elements with the following options:

* To search by name, enter the full or partial element name into **Search elements**.
* To filter based on the person who added the element, select **Added by**, then select the name of the team member.
* To filter by element type, select **Type**, then select **API**, **Collection**, or **Workspace**.
* To filter based on element tags, select **Tags**, then select the tags. Learn more about adding tags to [collections](/docs/collections/using-collections/#tagging-a-collection), [APIs](/docs/designing-and-developing-your-api/managing-apis/#tagging-apis), and [workspaces](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#tagging-a-workspace).
* To sort elements, select the **Sort by** dropdown list. Options are:
    * **Recently added** - Sort elements by the date they were added.
    * **A to Z** - Sort elements alphabetically.
    * **Trending this week** - Sort elements by your team's usage this week.
    * **Trending this month** - Sort elements by your team's usage this month.
```

Reviewer: @akinard-postman I think a few details aren't exactly right in the previous version. What do you think about these suggestions?

Author: I like this suggestion, thank you!

---

> **[The Private API Network is available on Postman Enterprise plans.](https://www.postman.com/pricing)**

The _Private API Network_ provides a [central directory](https://www.postman.com/api-platform/api-catalog/) of workspaces, collections, and APIs your team uses internally. Your Postman team can access these resources and start using them right away. By using the Private API Network, you can enable developers across your organization to discover, consume, and track API development in one place.

Workspaces, collections, and APIs in the Private API Network are visible to logged-in users who are on your Postman team. Users who aren't on your team can't find or access these resources. [Super Admins](/docs/collaborating-in-postman/roles-and-permissions/#team-roles), [API Network Managers](/docs/collaborating-in-postman/roles-and-permissions/#network-roles), and [Folder Managers](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) can manage all folders and elements in your Private API Network.

<img alt="Private API Network overview" src="https://assets.postman.com/postman-docs/v10/private-api-network-overview-v10-3.jpg"/>

## Contents

> * [Navigating the Private API Network](#navigating-the-private-api-network)
> * [Searching, filtering, and sorting](#searching-filtering-and-sorting)
> * [Reviewing details about elements](#reviewing-details-about-elements)
> * [Sharing folders and elements](#sharing-folders-and-elements)

Reviewer: ```suggestion
* [Navigating the Private API Network](#navigating-the-private-api-network)
* [Searching, filtering, and sorting](#searching-filtering-and-sorting)
    * [Searching for folders and elements in the network](#searching-for-folders-and-elements-in-the-network)
    * [Searching for elements in a folder](#searching-for-elements-in-a-folder)
* [Reviewing details about elements](#reviewing-details-about-elements)
* [Sharing folders and elements](#sharing-folders-and-elements)
```

  - type: link
    name: "Introducing an API to manage your Private API Network with more automation"
    url: "https://blog.postman.com/introducing-api-to-manage-your-private-api-network-with-automation/"
  - type: link
    name: "Simplifying API distribution and evaluation with the Private API Network"
    url: "https://blog.postman.com/simplifying-api-distribution-and-evaluation-with-the-private-api-network/"
  - type: subtitle
    name: "Case Studies"
  - type: link
    name: "ChargeHub plans to improve internal discovery with a Private API Network"
    url: "https://www.postman.com/case-studies/chargehub/"

---

> **[The Private API Network is available on Postman Enterprise plans.](https://www.postman.com/pricing)**

The _Private API Network_ provides a [central directory](https://www.postman.com/api-platform/api-catalog/) of workspaces, collections, and APIs your team uses internally. Your Postman team can access these resources and start using them right away. By using the Private API Network, you can enable developers across your organization to discover, consume, and track API development in one place.

> Workspaces, collections, and APIs in the Private API Network are visible to logged-in users who are on your Postman team. Users who aren't on your team can't find or access these resources. [Super Admins](/docs/collaborating-in-postman/roles-and-permissions/#team-roles), [API Network Managers](/docs/collaborating-in-postman/roles-and-permissions/#network-roles), and [Folder Managers](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) can manage all folders and elements in your Private API Network.

Reviewer: ```suggestion
Workspaces, collections, and APIs in the Private API Network are visible to logged-in users who are on your Postman team. Users who aren't on your team can't find or access these resources.

[Super Admins](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) and [API Network Managers](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) can manage all folders and elements in your Private API Network. You can assign team members the [Folder Manager role](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) at the folder level, giving them permission to manage specific folders and the elements in them.
```
